README.md

```diff
@@ -56,6 +57,20 @@ The Avalanche destination uses Azure Blob Storage to stage files before loading.
 - `tenantId` also called "Directory Id"
 - `clientSecret` provided by Azure Active 
 
+External authentication:
+
+```
+"externalAuth": {
+  "authId": String,
+  "userinfoUri": String, 
+  "userinfoUidField": String
+}
+```
+
+- `authId`: identifier obtained from the quasar implementation representing the externally obtained bearer token
+- `userinfoUri`: the uri which can be queried using the access token obtain with the `authId` identifier to obtain the UID to be used in the JDBC string
+- `userinfoUidField`: the field name that corresponds to a field on the `userinfoUri` response. The value of the field under this name will be inserted into the JDBC string as the UID.
```

Comment:
Just curious, so this is not always the same?

Reply:
So there _is_ an `email` field on `userinfo`, but in one of the cases it was null. The email was still on the payload, but in a different field (`sub`). It made me question how much we can make assumptions with regards to where that data will be, especially given that google SSO is to enter the mix, and Azure could in the future as well. 

It may well be that this can be removed in the future (maybe the `null` was just a dev environment thing), but being able to vary this is useful at least for the time being.

One thing I could do is add a default for it of `email`. Then we can stop caring if the problem solves itself in the future.

Reply:
I am fine leaving this as is. W.r.t. defaults it's easier to do that in the front end.